src/analysis/inner_product_space/basic.lean

```diff
@@ -1885,6 +1885,7 @@ open_locale direct_sum
 The simple way to express this concept would be as a condition on `V : ι → submodule 𝕜 E`.  We
 We instead implement it as a condition on a family of inner product spaces each equipped with an
 isometric embedding into `E`, thus making it a property of morphisms rather than subobjects.
+The connection to the subobject spelling is shown in `orthogonal_family_iff_pairwise`.
```

Comment:
Oh hmm, if you've already split out orthogonality, why is this still here?

Reply:
I left behind the orthogonal family stuff since it's not about submodules at all.

Reply:
I'm not opposed to a further split / reshuffle, but I think it's out of scope for this PR.